workers/data_refinery_workers/processors/array_express.py

```diff
@@ -23,16 +23,23 @@ def cel_to_pcl(kwargs: Dict):
     # Array Express processor jobs have one batch per job.
     batch = kwargs["batches"][0]
 
-    from_directory = utils.ROOT_URI + "raw/" + batch.internal_location
-    target_directory = utils.ROOT_URI + "processed/" + batch.internal_location
+    raw_file = os.path.join(utils.ROOT_URI, "raw", batch.internal_location, batch.name)
+
+    target_directory = os.path.join(utils.ROOT_URI, "processed", batch.internal_location)
     os.makedirs(target_directory, exist_ok=True)
-    new_name = batch.name + "." + batch.processed_format
 
-    ro.r('source("/home/user/r_processors/process_cel_to_pcl.R")')
-    ro.r['ProcessCelFiles'](
-        from_directory,
-        "Hs",  # temporary until organism handling is more defined
-        target_directory + new_name)
+    processed_file = os.path.join(target_directory, batch.name.split(".")[0])
+    processed_file = processed_file + "." + batch.processed_format
+
+    # It's necessary to load the foreach library before calling SCANfast
+    # because it doesn't load the library before calling functions
+    # from it.
+    ro.r("library('foreach')")
```

Comment:
LOL. Well, I guess this is still a net win if `SCAN.UPC` is well maintained and we don't have to write our own `process_cel_to_pcl.R` now.